packages/plugins/mocha-tests/src/lib/index.js

```diff
@@ -117,16 +129,12 @@ class MochaTestRunner {
               if (!compiledContracts[contract.className]) {
                 compiledContracts[contract.className] = {};
               }
+              instance.options.from = accounts[0];
+              instance.options.gas = 900000;
```

Comment:
I know this is kinda hack-ish. We'd need to go back to 4.0 to see what was done there to make it so that Teller tests didn't need to manually pass the gas